functions/calculate-score/index.js

```diff
@@ -0,0 +1,20 @@
+'use strict'
+
+const { performance } = require('perf_hooks')
+
+const v1 = require('./v1')
+
+module.exports = async function (context) {
+  const start = performance.now()
+  try {
+    await v1.process(context)
```

Comment:
Usually, we return some meta-data from the process.  This could be the number of scores processed, the number of checks processed, the number of pupils processed?  Potentially some interesting stuff.

I wouldn't mind seeing a relatively verbose message for this one in the log.

Reply:
@jon-shipley I doubt what you are sugggesting can be achieved since the store procedure will conclude at a later time from the function completion time.
Does this answer your question?

Reply:
@GeorgeChatzigiannis.  The SP can easily return some data to the client that would `await` on it.  We do exactly this for the check creation.   I'm happy to help.   Give me a shout when the SP is further along.

Reply:
@GeorgeChatzigiannis you can use explicit return values in stored procedures, or just do a SELECT.  will be easier to consume in tedious if you do a SELECT.

Reply:
with regard to sp completion and function completion - that is a rather scary statement and forward view on this piece of work.  needs discussion